src/pages/login.ts

```diff
@@ -25,8 +25,7 @@ export class LoginPage {
         conn.accessToken
       }&retURL=${encodeURIComponent(POST_LOGIN_PATH)}`,
       {
-        // should have waited at least 500ms for network connections, redirects should probably have happened already
-        waitUntil: ['load', 'networkidle2'],
+        waitUntil: 'load',
```

Comment:
As this got more simple and 'load' is the default, we can remove the `waitUntil` option here and in all other places.

https://playwright.dev/docs/api/class-page#page-goto-option-wait-until